content/configuration/tasks-deploy.md

```diff
@@ -185,6 +185,21 @@ specification:
             memory: 500Mi
 ```
 
+## Replicas and Redundancy 
+
+{{< note title="Note" >}}
+For background, a pod comprises a Kubernetes service and container(s). A replica is copy of a pod running on a different node, where a node is a worker machine within a cluster.
```

Comment:
> "What does that mean?"

Its just trying to establish what a pod is so that replicas can be better explained. Can you suggest a rewording that I can work with? Or is it the use of the word "comprise"? I was trying to avoid using "contains" as it sounds like container.